app/lib/common/utilities/email_sender.dart

```diff
@@ -1,40 +1,46 @@
+import 'dart:io';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter_email_sender/flutter_email_sender.dart';
 import 'package:flutter/material.dart';
 
 class EmailSender {
-  Future<void> send(
-      String emailBody, String eSubject, String eRecipients) async {
+  String emailResponse;
```

Comment:
Seems odd to have the email response be a field of the class, it should have been defined in `send` like before. Something to fix in the future.